### PR TITLE
docs: update installation instructions for tauri v1 based on crabnebula docs

### PR DESCRIPTION
**Pull request by pacholoamit**
# Description

The current README for tauri v1 installation doesn't work quite right. I was fortunate enough to find the docs on the crabnebula site so I just updated the docs for tauri v1.

# References

- https://docs.crabnebula.dev/devtools/get-started/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,16 +21,35 @@ Ensure you have [Tauri](https://tauri.app/v1/guides/getting-started/setup/) set
...
+        .run(tauri::generate_context!("./tauri.conf.json"))
+        .expect("error while running tauri application");
+}
+```
+
+> Note: You should disable devtools in production builds, as it bloats your app unnecessarily and poses a possible security risk.
+
+And that’s it! If you run your app now (cargo tauri dev), you will notice the following new output in your terminal:
+
+[Terminal input](https://docs.crabnebula.dev/_astro/stdout.08yrN9L2_Z1wy72E.webp)
```
Comment (johann-crabnebula) [inline README.md:51]: This asset should be properly copied and added to the repository, since the URL might change between our docs deployments.

**Review by johann-crabnebula — CHANGES_REQUESTED** [review]
Hey @pacholoamit 👋 
Thanks for taking interest in DevTools. We appreciate any community input we get. First of all, we are sorry for any inconvenience caused by the README not being up to date. 

Thanks for providing an update! There is a little thing that we would like to see fixed before approving.

File: README.md (hunk 1 above)
Comment (pacholoamit) [reply README.md]: That makes sense. I've added the asset to the `docs` directory in the repo. Thanks!

**Review by johann-crabnebula — APPROVED** [review]
Thanks for the quick adjustments.

Now it matches the docs!

Happy to merge this :)

**Pull request merged**